### ISSUE TITLE
ModuleNotFoundError: No module named 'bridge.constanst'

### ISSUE THREAD

**Issue by schmir-at-bb** [trustlines-protocol/blockchain#307]
```
..lines/protocol/blockchain/tools/bridge (git)-[configure-logging] % tlbc-bridge --help
Traceback (most recent call last):
  File "/home/bbralf/trustlines/venv/bin/tlbc-bridge", line 11, in <module>
    load_entry_point('tlbc-bridge', 'console_scripts', 'tlbc-bridge')()
  File "/home/bbralf/trustlines/venv/lib/python3.6/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/bbralf/trustlines/venv/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2843, in load_entry_point
    return ep.load()
  File "/home/bbralf/trustlines/venv/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2434, in load
    return self.resolve()
  File "/home/bbralf/trustlines/venv/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2440, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/bbralf/trustlines/protocol/blockchain/tools/bridge/bridge/main.py", line 17, in <module>
    from bridge.constanst import TRANSFER_EVENT_NAME
ModuleNotFoundError: No module named 'bridge.constanst'

```

Looks like this has been introduced in abb3256596d98d99ffe5da4391c40d3cf28b3188 (/cc @jannikluhn)

**Comment by jannikluhn** [comment]
Fixed in #290 

Shouldn't CI have catched that?

**Comment by schmir-at-bb** [comment]
> Fixed in #290

not really, it would have been nice if you opened a PR to fix that single issue in case you've been aware of it. I've merged #308 without any review.

> 
> Shouldn't CI have catched that?

Apparently the tests do not import the bridge.main module.
